src/Components/IntroSlider.tsx

```diff
@@ -0,0 +1,137 @@
+import { useNavigation } from "@react-navigation/native";
+import React, { useRef, useState } from "react";
+import { View, Text, Button, Image, StatusBar, Platform } from "react-native";
+import AppIntroSlider from "react-native-app-intro-slider";
+import { SafeAreaView } from "react-native-safe-area-context";
+import {
+  heightPercentageToDP as h2dp,
+  widthPercentageToDP as w2dp,
+} from "react-native-responsive-screen";
+
+const slides = [
+  {
+    key: "slide1",
+    title: "Welcome to My App",
+    text: "Volunteer at an event, donate food, or become a driver.",
+    image: require("../../assets/images/volunteerscollecting.png"),
+  },
+  {
+    key: "slide2",
+    text: "Keep track of your donation history.",
+    image: require("../../assets/images/onlineTaxes.png"),
+  },
+  {
+    key: "slide3",
+    text: "Easily share your volunteer successes with your social network.",
+    image: require("../../assets/images/shareAdvertising.png"),
+  },
+];
+
+const IntroSlider = () => {
+  const [currentSlideIndex, setCurrentSlideIndex] = useState(0);
+
+  const sliderRef: any = useRef(null);
+
+  const navigation: any = useNavigation();
+  const renderItem = ({ item }: any) => (
+    <>
+      <SafeAreaView style={{ height: "100%" }}>
+        <StatusBar backgroundColor="auto" barStyle={"dark-content"} />
+        <View
+          style={{
+            alignItems: "center",
+            justifyContent: "space-around",
+            height: "100%",
+          }}
+        >
+          <Image source={item.image} />
```

Comment:
@Shoebmalik007 , can you please add null checks here as because of this we might face crash issues